src/components/HeaderBar.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactComponent as KHLogo } from '../assets/logo.svg';
+import { ReactComponent as DrawerHamburger } from '../assets/drawerHamburger.svg';
+import { Link } from 'react-router-dom';
+
+export default function HeaderBar() {
+
+	return (
+		<div className="flex justify-between w-full p-8 shadow-headerbar mb-3">
```

Comment:
Make it p-4 to make it a bit closer to the design presented in the figma.

Reply:
I had it as p-4 originally which seemed too close on my monitor, but I can certainly change it back.